config/environments/development.rb

```diff
@@ -13,7 +13,7 @@
   config.eager_load = false
 
   # Show full error reports.
-  config.consider_all_requests_local = true
+  config.consider_all_requests_local = false
```

Comment:
Please set this value back to TRUE before you push your code to merge it with the master.